feature/bgp/prefixlimit/ate_tests/bgp_prefix_limit_test/README.md

```diff
@@ -6,7 +6,7 @@ BGP Prefix Limit
 
 ## Procedure
 
-*   Configure BGP session between ATE port-1 and DUT port-1.
+*   Configure eBGP session between ATE port-1 and DUT port-1,with an Accept-route all import-policy/export-policy under the neighbor AFI/SAFI.
```

Comment:
@arvbaska1  The code says it's under peer-group, please help update this README if you agree. Same applies to other changes in this PR. Thanks. 

cc: @guoshiuan

Reply:
Yes, will update the Readme for the tests.